src/email/index.ts

```diff
@@ -43,4 +51,49 @@ export async function sendEmail(
     subject,
     html,
   });
-}
\ No newline at end of file
+}
+
+export class EmailClient {
+  async sendUserInvite(to: string, props: { inviteUrl: string }) {
+    const html = await render(UserInviteTemplate({ ...props }));
```

Comment:
don't need to spread, can pass as is
```suggestion
    const html = await render(UserInviteTemplate(props));
```